docs/README.md

```diff
@@ -0,0 +1,256 @@
+# Microsoft Azure SDK for Ruby
+
+Microsoft Azure SDK for Ruby has been deprecated. It will be in maintenance mode for a period of time. This doc instructs how to use Azure REST API as a replacement. By the way, it is highly recommanded you to use GA SDKs
+* [Python](https://github.com/Azure/azure-sdk-for-python)
+* [Java](https://github.com/Azure/azure-sdk-for-java)
+* [JavaScript](https://github.com/Azure/azure-sdk-for-js)
+* [.Net](https://github.com/Azure/azure-sdk-for-net)
+
+You can also find other SDKs in [Azure SDK](https://github.com/Azure/azure-sdk).
+
+# Azure REST API
+
+[Azure REST API Reference](https://docs.microsoft.com/en-us/rest/api/azure/)
+
+
+## Authentication
+
+The first step to using the SDK is authentication and permissions. For people unfamilar with Azure this may be one of
+the more difficult concepts. For a reference on setting up a service principal from the command line see
+[Authenticating a service principal with Azure Resource Manager](http://aka.ms/cli-service-principal) or
+[Unattended Authentication](http://aka.ms/auth-unattended). For a more robust explanation of authentication in Azure,
+see [Developer’s guide to auth with Azure Resource Manager API](http://aka.ms/arm-auth-dev-guide).
+
+After creating the service principal, you should have three pieces of information, a client id (GUID), client secret
+(string) and tenant id (GUID) or domain name (string).
+
+# Usage of ms_rest_azure gem
+
+The ms_rest_azure gem is a Azure rest client library used by Azure SDK for Ruby. It relies on faraday and concurrent-ruby. 
+
+## Install
+
+You can install the ms_rest_azure gem with the following command:
+
```

Comment:
consider moving "The ms_rest_azure gem is a Azure rest client library used by Azure SDK for Ruby. It relies on faraday and concurrent-ruby. " into the Install section